src/genie/libs/parser/iosxe/tests/ShowVrrp/cli/equal/golden_output1_expected.py

```diff
@@ -0,0 +1,41 @@
+expected_output = { 
+     {
+    "interface": {
+        "GigabitEthernet1": {
+            "group_number": {
+                "Group 100": {
+                    "auth_text": '"hash"',
+                    "device_advertise_interval": "1.000",
```

Comment:
Let's have unit `sec` in key name and value is float.
```suggestion
                    "advertise_interval_secs": 1.000,
```

Reply:
confirmed the change